cmd/eksctl-anywhere/cmd/upgradecluster.go

```diff
@@ -65,7 +66,8 @@ func init() {
 	flags.MarkRequired(createClusterCmd.Flags(), flags.ClusterConfig.Name)
 }
 
-func (uc *upgradeClusterOptions) upgradeCluster(cmd *cobra.Command) error {
+// nolint:gocyclo
```

Comment:
`upgradeCluster` is already a high cyclomatic complexity function. If we take validation out, what remains are still above 10 cyclomatic complexity, plus we need to pass `clusterSpec` and `clusterConfig` from validation func or generate it again. So adding nolint here